packages/codegen/generate.server.utils.js

```diff
@@ -51,20 +49,22 @@ function _throwIfError (context, errors, data, errorMessage, errorMapping) {
          *    })
          *
          */
+        const fields = {
+            code: GQLErrorCode.INTERNAL_ERROR,
+            type: 'SUB_GQL_ERROR',
```

Comment:
Same here, what is `sub_gql_error`?

Reply:
It's internal type for cases GQL call -> call other GQL -> error. Then we need to save this error stack to show stacked trace.

Reply:
Got you. I guess naming here can be more clear...

Because in my head `SUB_GQL_ERROR` was more related to GQLError (its sub-type in my head).

Maybe `SUB_QUERY_EXECUTION_ERROR`? Or `SUB_QUERY_ERROR`....Hmmm, idk

Reply:
Should we also move it to const somewhere, so it can be accessible in tests / changed easily?

Reply:
I don't want to use QUERY / MUATION word because it's also confusing.

Reply:
I'll add more detaild and will use one global variable for that. It will help to understand whats going on